### DISCUSSION TITLE
Conversation Support

### DISCUSSION THREAD

**Discussion by irazasyed** [telegram-bot-sdk/telegram-bot-sdk#69] [Ideas]
This will most likely be an extension of this SDK.

Some references to consider from the community.

- https://github.com/irazasyed/telegram-bot-sdk/issues/533

**Comment by alies-dev** [comment]
@irazasyed 
Dialogs/Conversations are developed as a separate package https://github.com/koot-labs/telegram-bot-dialogs (works on v3.1+) (I'm a maintainer).

I think it's good to have such functionality as a separate  package (plugin)

**Reply by pullaf** [reply]
Any chance you could update that library to work with Laravel 10? It's the only thing keeping me locked on 9
